### DISCUSSION TITLE
Add shortcut to settings in the energy dashboard

### DISCUSSION THREAD

**Discussion by aahlborg** [home-assistant/frontend#11665] [Dashboard]
I was setting up my electricity usage sensor and I tried to add it to the energy dashboard but it didn't show up. Somehow I managed to add it by typing its name but it showed up without a name so I deleted it again. After this the initial setup disappeared and the energy dashboard was just showing an empty graph. I couldn't figure out how to get to the settings but after an hour of googling and clicking around I found it in the dashboards section.

It would be helpful if there was a menu button on the energy dashboard that contained a shortcut to the energy settings.

**Comment by ludeeus** [comment]
There is an even easier way to get there.
1) Hit "c" on your keyboard to bring up the quickbar
2) Search for it and click the entry for it

![image](https://user-images.githubusercontent.com/15093472/153700259-9c3810d2-8115-4bc2-994a-460f7e9d83fc.png)
